update_prevalence.py

```diff
@@ -696,10 +697,12 @@ def main() -> None:
 
         # Test positivity per US county and state
         for item in parse_json(cache, CANRegionSummary, CANRegionSummary.COUNTY_SOURCE):
+            assert type(item.fips) is int, "Expected item.fips to be int but got {}".format(type(item.fips))
             if item.fips not in data.fips_to_county:
                 # Ignore e.g. Northern Mariana Islands
+                print(f"ignoring unknown county fips {item.fips}")
```

Comment:
I think there are several of these in normal operation -- not sure we should keep this printout?

Reply:
There are only 2 now that I've fixed the <5 digit FIPS code bug. Before the long list of these was (I think) every county in the first 10 states alphabetically (Alabama - Connecticut)